contrib/kubernetes/manifests/registry-gcs-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gcs-keyfile
+type: Opaque
+data:
+  key.json: {base64 encode of the service account json file data}
+  bucket: {base64 encode of the buket name}
```

Comment:
GCS doesn't have the concept of region and Azure doesn't use the term bucket.So, i feel it is better to have the complete configuration in the secrets.